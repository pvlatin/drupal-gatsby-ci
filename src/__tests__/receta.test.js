// import React from 'react'
// import renderer from 'react-test-renderer'
// import Receta from '../components/receta'

// describe('Receta', () => {
//   it('renders correctly', () => {
//     const tree = renderer.create(<Receta />).toJSON()
//     expect(tree).toMatchSnapshot()
//   })
// })