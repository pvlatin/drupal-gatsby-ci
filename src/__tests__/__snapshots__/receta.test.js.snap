// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Receta renders correctly 1`] = `
<div>
  <div
    className="c-display-receta"
  />
</div>
`;
